### DISCUSSION TITLE
wrong indoor temperature shown

### DISCUSSION THREAD

**Discussion by wasn-eu** [blakadder/nspanel#5] [Q&A]
Hi

i have flashed my NSPanel with the tasmota_nspanel image.
Added it to HomeAssistant without problems.
Thank you for your great doc and work.

the only issue i have is that the indoor temperature is wrong.
It is allways showing -5C.
i allways get this
` tele/nspanel/SENSOR = {"Time":"2021-12-30T01:17:07","ANALOG":{"Temperature1":-5.4},"ESP32":{"Temperature":68.3},"TempUnit":"C"}
`
the ESP32 Temp is changing

what can i do to get a right indoor temperature?

**Comment by blakadder** [comment]
nspanel image is being built with the wrong base for the past few days, waiting on a fix. use tasmota32.bin in the meanwhile
